docs/src/content/docs/de/guides/components.mdx

```diff
@@ -0,0 +1,199 @@
+
+Da Starlight auf Astro basiert, kannst du Komponenten, die mit einem beliebigen [unterstützten UI-Framework (React, Preact, Svelte, Vue, Solid, Lit und Alpine)](https://docs.astro.build/de/core-concepts/framework-components/) erstellt wurden, in deine MDX-Dateien aufnehmen.
+Erfahre mehr über [Verwendung von Komponenten in MDX](https://docs.astro.build/de/guides/markdown-content/#using-components-in-mdx) in den Astro-Dokumenten.
+
+### Kompatibilität mit Starlights Stilen
+
+Starlight wendet Standardstile auf deinen Markdown-Inhalten an, zum Beispiel das Hinzufügen von Abständen zwischen Elementen.
+Wenn diese Stile mit dem Erscheinungsbild deiner Komponente in Konflikt stehen, setze die Klasse `not-content` auf deine Komponente, um sie zu deaktivieren.
+
+```astro
+---
+// src/components/Beispiel.astro
+---
+
+<div class="not-content">
+	<p>Not impacted by Starlight’s default content styling.</p>
+</div>
+```
+
+## Eingebaute Komponenten
+
+Starlight bietet einige standardmäßig eingebaute Komponenten für gängige Dokumentationszwecke.
+Diese Komponenten sind aus dem Paket `@astrojs/starlight/components` verfügbar.
+
+### Registerkarten
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Mit den Komponenten `<Tabs>` und `<TabItem>` kannst du eine Oberfläche mit Registerkarten anzeigen.
+Jedes `<TabItem>` muss ein `label` haben, welches dem Benutzer angezeigt wird.
+
+```mdx
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs>
+	<TabItem label="Sterne">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Monde">Io, Europa, Ganymed</TabItem>
+</Tabs>
+```
+
+Der obige Code erzeugt die folgenden Registerkarten auf der Seite:
+
+<Tabs>
+	<TabItem label="Sterne">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Monde">Io, Europa, Ganymed</TabItem>
+</Tabs>
+
+### Karten
+
+import { Card, CardGrid } from '@astrojs/starlight/components';
+
+Mit der Komponente `<Card>` kannst du Inhalte in einer Box darstellen, die den Stilen von Starlight entspricht.
+Packe mehrere Karten in die `<CardGrid>` Komponente ein, um Karten nebeneinander darzustellen, wenn genügend Platz vorhanden ist.
+
+Eine `<Card>` benötigt einen `title` und kann optional ein `icon` Attribut enthalten, welches auf den Namen [eines von Starlights eingebauten Icons](#alle-icons) gesetzt ist.
+
+```mdx
+import { Card, CardGrid } from '@astrojs/starlight/components';
+
+<Card title="Schau mal hier">
+	Interessanter Inhalt, den du hervorheben möchtest.
+</Card>
+
+<CardGrid>
+	<Card title="Sterne" icon="star">
+		Sirius, Wega, Betelgeuse
+	</Card>
+	<Card title="Monde" icon="moon">
+		Io, Europa, Ganymed
+	</Card>
+</CardGrid>
+```
+
+Der obige Code erzeugt folgendes auf der Seite:
+
+<Card title="Schau mal hier">
+	Interessanter Inhalt, den du hervorheben möchtest.
+</Card>
+
+<CardGrid>
+	<Card title="Sterne" icon="star">
+		Sirius, Wega, Betelgeuse
+	</Card>
+	<Card title="Monde" icon="moon">
+		Io, Europa, Ganymed
+	</Card>
+</CardGrid>
+
+:::tip
```

Comment:
Hier wird in der Preview als Überschrift `Tip` anstatt `Tipp` angezeigt. Ist das evtl. noch ein String, der irgendwo übersetzt werden muss?

Reply:
da kann man glaub ich einfach eckige Klammern verwenden:

:::tip[Tipp]
:::